### DISCUSSION TITLE
Failed to add UTM cydia repo to Sileo

### DISCUSSION THREAD

**Discussion by 185264646** [utmapp/UTM#6577] [UTM Help (iOS)]
**Describe the issue**
When adding https://cydia.getutm.app to the repo list of Sileo, it failed with
```
Error: Didn't find architectures DPKGArchitectures() in https://cydia.getutm.app/Release
Error: Could not find release file for https://cydia.getutm.app/
```

**Configuration (required)**
* UTM Version: Not Installed
* OS Version: iOS 16.1
* Device Model: iPad Pro (gen 4)
* Is it jailbroken (name jailbreak used)? Dopamine 2.1.5
* How did you install UTM? just follow the official guide (jail-broken)

** Extra Info **
I've found some similar reports. Maybe this can help.

https://github.com/akemin-dayo/AppSync/issues/163

**Comment by ImBlu3** [comment]
Install the trollstore version, as you are using dopamine, you should already have trollstore, so just install the .ipa file that way

**Comment by 185264646** [comment]
> Install the trollstore version, as you are using dopamine, you should already have trollstore, so just install the .ipa file that way

The wiki page said JIT (i.e. `dynamic-codesigning` entitlement) is not supported by TrollStore. I've tested that UTM.ipa can not be opened in anyway from TrollStore (by clicking `Open With JIT` or `Open`, it returns to desktop immediately and give a dialog `Failed to open com.utmapp.UTM`) even on a jailbroken device. UTM-SE.ipa works but is very slow. So i think it should be installed from Sileo if JIT is desired.

**Comment by n-WN** [comment]
> > Install the trollstore version, as you are using dopamine, you should already have trollstore, so just install the .ipa file that way
> 
> The wiki page said JIT (i.e. `dynamic-codesigning` entitlement) is not supported by TrollStore. I've tested that UTM.ipa can not be opened in anyway from TrollStore (by clicking `Open With JIT` or `Open`, it returns to desktop immediately and give a dialog `Failed to open com.utmapp.UTM`) even on a jailbroken device. UTM-SE.ipa works but is very slow. So i think it should be installed from Sileo if JIT is desired.

UTM.HV.ipa has been installed on iPhone using TrollStore2. Virtualization is not supported when opened after installation (it is expected to be useful only for the M series). The JITSTREAMER option in the Settings in UTM can be turned on, but I didn't go to the next step

**Comment by n-WN** [comment]
>*gen 4

gen 4? I think Apple’s confusing naming of iPad Pro would be more appropriate to refer to the chip.

**Comment by osy** [comment]
This error is because our repo does not support rootless. If you are using a rootless jailbreak, you need to use trollstore because I don't think appsync unified works.
